platforms/jvm/war/src/main/java/org/gradle/api/tasks/bundling/War.java

```diff
@@ -97,7 +97,7 @@ public CopySpec getWebInf() {
      * @param configureClosure The closure to execute
      * @return The newly created {@code CopySpec}.
      */
-    public CopySpec webInf(@DelegatesTo(CopySpec.class) Closure configureClosure) {
+    public CopySpec webInf(@DelegatesTo(CopySpec.class) Closure<CopySpec> configureClosure) {
```

Comment:
I worry that this could be a source-incompatible change -- though I'm not completely sure. I'd expect this to remain binary compatible though.

There are a lot of examples of `@SuppressWarnings("rawtypes")` in similar places elsewhere.

Reply:
I think if this was source-incompatible, there would be other problems.  But it's not important to do this now, I was just clearing warnings.  I'll revert these.